index.html

```diff
@@ -0,0 +1,201 @@
+                                Cook the burgers on a preheated barbecue or griddle for 5-6 minutes on each side. While the second side is cooking, lay a slice of cheese on top to melt slightly (if using).
+                                Meanwhile, lightly toast the cut-sides of the buns on the barbecue. Fill with the lettuce, burgers and tomato slices. Serve with ketchup, if you like.
+                            </p>
+                        </span>
+                    </p>
+                    <div class="buttons">
+                        <button onclick="myFunction()" id="myButton">View More</button>
+                        <!-- <i class="fa-regular fa-heart"></i> -->
+                        <a href="https://kit.fontawesome.com/a462069d14.js" crossorigin="anonymous"><i class="fa-regular fa-heart"></i></a>
+                    </div>
+            </div>
+            <div>
+                <img src="pexels-dzenina-lukac-1583884.jpg" alt="crispy chips">
+                <h2>Dried Crunchy Potato Chips</h2>
+                <h2>KSH 480.00</h2>
+                <p>Plain dried chips, crunchy and salted</p>
+                <hr>
+                    <p>
+                        Ingredients and Instructions<span id="dots">...</span>
+                        <span id="more">
+                            <h4>Ingredients</h4>
+                                <ul>
+                                    <li>Potatoes</li>
+                                    <li>Canola vegetable oil</li>
+                                    <li>Sunflower oil</li>
+                                    <li>Maltodextrin</li>
+                                    <li>Natural flavours</li>
+                                    <li>Salt</li>
+                                    <li>Malic acid</li>
+                                    <li>Vinegar</li>
+                                </ul>
+                                <h4>Instructions/Method</h4>
+                                <p>
+                                    Receive, grade and sort potatoes<br>
+                                    Clean, wash and destone potatoes<br>
+                                    Peel the Potatoes<br>
+                                    Slice and trim the peeled potatoes to size<br>
+                                    Fry the sliced potatoes in enough oil<br>
+                                    Season and salt to taste<br>
+                                    Cool and dry excess oil<br>
+                                    Serve to satisfaction<br>
+                                </p>
+                        </span>
+                    </p>
+                    <div class="buttons">
+                        <button onclick="myFunction()" id="myButton">View More</button>
+                        <!-- <i class="fa-regular fa-heart"></i> -->
+                        <a href="https://kit.fontawesome.com/a462069d14.js" crossorigin="anonymous"><i class="fa-regular fa-heart"></i></a>
```

Comment:
remove href and replace with #

Reply:
> remove href and replace with #

done, and replaced the href link with an #